thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/completeness/checker/DataCompletenessScheduler.java

```diff
@@ -1,46 +1,29 @@
 package com.linkedin.thirdeye.completeness.checker;
 
-import java.util.ArrayList;
-import java.util.List;
 import java.util.concurrent.Executors;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
-
-import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.google.common.collect.Lists;
-import com.linkedin.thirdeye.client.ThirdEyeCacheRegistry;
-
 /**
  * The scheduler which will run periodically and schedule jobs and tasks for data completeness
  */
 public class DataCompletenessScheduler {
 
   private static final Logger LOG = LoggerFactory.getLogger(DataCompletenessScheduler.class);
 
-  private ScheduledExecutorService scheduledExecutorService;
+  private ScheduledExecutorService scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();;
   private DataCompletenessJobRunner dataCompletenessJobRunner;
   private DataCompletenessJobContext dataCompletenessJobContext;
-  private List<String> datasetsToCheck = new ArrayList<>();
-
-  public DataCompletenessScheduler(String datasetsToCheck) {
-    scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();
-    if (StringUtils.isNotBlank(datasetsToCheck)) {
-      this.datasetsToCheck = Lists.newArrayList(datasetsToCheck.split(","));
-    }
-  }
 
   public void start() {
-
     LOG.info("Starting data completeness checker service");
 
     dataCompletenessJobContext = new DataCompletenessJobContext();
-    dataCompletenessJobContext.setDatasetsToCheck(datasetsToCheck);
     dataCompletenessJobRunner = new DataCompletenessJobRunner(dataCompletenessJobContext);
 
-    scheduledExecutorService.scheduleWithFixedDelay(dataCompletenessJobRunner, 0, 15, TimeUnit.MINUTES);
+    scheduledExecutorService.scheduleWithFixedDelay(dataCompletenessJobRunner, 0, 5, TimeUnit.MINUTES);
```

Comment:
This shouldn't even be here. A config file would be a better place. Suggestions?